UnitySDK/Assets/ML-Agents/Scripts/DemonstrationRecorder.cs

```diff
@@ -71,14 +73,22 @@ public void WriteExperience(AgentInfo info)
             m_DemoStore.Record(info);
         }
 
+        public void Close()
```

Comment:
Should this happen on disable instead? Need it for testing to make sure the write stream is flushed,

Reply:
would you move the initialization to OnEnable?

Reply:
It's already done lazily in Update()

Reply:
should it be set to `null` after it's closed?  Is there a use case for stopping recording during play mode and resuming?